src/components/views/voip/CallView.tsx

```diff
@@ -566,7 +617,33 @@ export default class CallView extends React.Component<IProps, IState> {
                         />
                     </div>
                 </div>
-                {holdTransferContent}
+                <div className="mx_CallView_holdTransferContent">{_t("Connecting")}</div>
+                {callControls}
+            </div>;
+        } else {
+            const containerClasses = classNames({
+                mx_CallView_video: true,
+            });
+
+            // TODO: Later the CallView should probably be reworked to support any
+            // number of feeds but now we can always expect there to be two feeds
```

Comment:
what does this code do if two video tracks are sent?

Reply:
It probably breaks :D I intentionally left this out as doing this well (my idea of how that would look would be [#16110](https://github.com/vector-im/element-web/issues/16110)) is whole another PR

Reply:
(for scrensharing we can actually do something quite simpler but I'd still like to keep that separate as this is quite hard to test now anyway)

Reply:
OK, but it looks like it might break by trying to shove all the video feeds into the container. If we do introduce more feeds later, older clients that don't support it need to do something sensible: just displaying one stream (ideally the user media one) is fine but trying to display them all & ending up with a broken layout will cause problems.

Reply:
True, I should have thought of that :facepalm: I'll rework it to be safer

Reply:
OK - if we're relying on undocumented behaviour of the js-sdk that probably deserves a comment.